sound/soc/sof/loader.c

```diff
@@ -86,7 +86,7 @@ static int get_cc_info(struct snd_sof_dev *sdev,
 }
 
```

Comment:
The changes look welcome, but summary line is a bit misleading. You state you remove uses of u32, but then the patch actually does a bit more... at least one change of int to size_t and removal of redundant variable Maybe one separate patch with non-u32 fixes?